### DISCUSSION TITLE
When moving a repo from personal account to organization co-pilot gets turned off

### DISCUSSION THREAD

**Discussion by anonrose** [community/community#64551] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have a personal license with co-pilot, when I moved a repository from my personal account to an organization I could no longer use co-pilot. I assume this is because the organization doesn't pay for co-pilot.

After remove the repository locally and re-cloning from the organization I could once again use co-pilot on that repo but the issue remains that moving projects doesn't maintain the ability to use co-pilot.

**Comment by Kaysuto** [comment]
Hello!

The issue is that the Copilot license is tied to a personal GitHub account, and does not carry over to organizations.

Some solutions to retain Copilot access when moving a repo to an organization:

- Upgrade your personal account to a GitHub Pro account, which includes a Copilot license usable in organizations you are a member of.
- Request the organization to purchase a Copilot license so all members can use it.
- Keep a copy of the repo in your personal account mirrored to the organization's repo, to maintain Copilot access on that copy.
- As you noticed, deleting and re-cloning the repo after moving to the organization re-enables Copilot, but not very convenient.
- Contribute to the organization's repo via forks from your personal account rather than directly.

It would be good if GitHub allowed retaining Copilot enabled when moving a repo between accounts, even if licenses differ.

Let me know if you have any other questions!

**Comment by anonrose** [comment]
@Kaysuto I do have a pro account so unsure if that's a solution but thank you for the you for the quick response. Is this documented some place?

**Comment by Kaysuto** [comment]
So you have a personal GitHub Pro account, this should allow you to use Copilot within the repositories of the organizations to which you belong. GitHub documents this here: https://docs.github.com/fr/copilot

The Copilot license is linked to your GitHub account, so as long as you contribute to the organization's repositories through your Pro account, you should be able to use Copilot.

However, if other members of the organization do not have a Pro account or a Copilot license, they will not have access to it. In this case, the organization should purchase a Copilot license so that all members can use it.

Feel free to ask if you have any other questions! I'm happy to have been able to clarify how Copilot licenses work between personal accounts and organizations.
